src/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+/opt/ros/melodic/share/catkin/cmake/toplevel.cmake
```

Comment:
I think you should avoid distributing this symlink with the project - catkin creates it automatically and this will only break builds in the future for people with different installs / versions.

Reply:
I removed this CMake file. Thanks for the heads up about the problem other people may run into with this file.